src/taco-cli/cli/platform.ts

```diff
@@ -170,6 +176,22 @@ class Platform extends KitComponentCommand {
                 break;
 
         }
+
+    }
+    private list(): Q.Promise<ICommandTelemetryProperties> {
+        return this.passthrough()
+            .then(function(telemetryProperties: ICommandTelemetryProperties): Q.Promise<ICommandTelemetryProperties> {
+                // The user has invoked 'taco platform list'
+                // We want to add an additional line including any remote platforms
+                return Settings.loadSettingsOrReturnEmpty()
+                    .then((settings: Settings.ISettings): void => {
+                        var remotePlatforms: string[] = Object.keys(settings.remotePlatforms || {});
+                        if (remotePlatforms.length > 0) {
+                            Logger.log(resources.getString("PlatformListRemotePlatforms", remotePlatforms.join(",")));
+                        }
+                    })
+                    .then(() => telemetryProperties);
```

Comment:
Should we update the telemetry properties with anything from here? You can also just return telemetryProperties in the previous then rather than chaining a new one.